### PR TITLE
BlitzGateway 5.3.0

### PR DESCRIPTION
**Pull request by will-moore**
Tidy and reorganise BlitzGateway documentation page.
Didn't need to add any (much) different for 5.3.0, but I've improved the readability of the page by re-ordering some sections and removing some little bits. Fixing odd bits like "see above" at the start of the page etc.

Just needs a read-through to check for any similar glitches.

https://trello.com/c/PEvlrmHX/36-document-breaking-api-changes-for-devs-add-new-examples

### CODE REVIEW

File: omero/developers/PythonBlitzGateway.txt [hunk 1]
```diff
@@ -1,19 +1,70 @@
...
 
-The Sphinx-generated documentation of methods provided by OMERO Gateway
-is available showing :pythondoc:`wrapper classes <omero/omero.gateway.html>`.
+This page provides some background information on the OMERO Python client
+'gateway' (omero.gateway module).
+
+The Blitz Gateway is a Python client-side library that facilitates working
+with the OMERO API, handling connection to the server, loading of data objects
+and providing convenience methods to access the data. It was originally
+designed as part of the |OmeroWeb| framework, to provide connection and data
```
Comment (simleo) [inline omero/developers/PythonBlitzGateway.txt]: Remove "framework" (`|OmeroWeb|` already renders as "OMERO.web framework")

File: omero/developers/PythonBlitzGateway.txt [hunk 2]
```diff
@@ -1,19 +1,70 @@
...
+
+
+
+Connection wrapper
+""""""""""""""""""
+
+The BlitzGateway class (see :pythondoc:`API of development code <omero/omero.gateway.html#omero.gateway._BlitzGateway>`)
+is a wrapper for the OMERO client and session objects. It provides
+various methods for connecting to the OMERO server, querying the status
+or context of the current connection and as a starting point for
```
Comment (simleo) [inline omero/developers/PythonBlitzGateway.txt]: Remove "as"

File: omero/developers/PythonBlitzGateway.txt (hunk 1 above)
Comment (dominikl) [reply omero/developers/PythonBlitzGateway.txt]: Same in line 139 (not introduced by the PR but could be fixed on that occasion, too).

**Comment by jburel** [comment]
``/home/travis/build/openmicroscopy/ome-documentation/omero/developers/whatsnew.txt:16: WARNING: Inline interpreted text or phrase reference start-string without end-string.``

**Comment by will-moore** [comment]
Added to What's New https://www.openmicroscopy.org/site/support/omero5.3-staging/developers/whatsnew.html 

Also updated https://www.openmicroscopy.org/site/support/omero5.3-staging/developers/Web.html
and https://www.openmicroscopy.org/site/support/omero5.3-staging/developers/Web/WebGateway.html 

Added note about App Config to https://www.openmicroscopy.org/site/support/omero5.3-staging/developers/Web/CreateApp.html

File: omero/developers/Web/CreateApp.txt [hunk 3]
```diff
@@ -164,6 +164,17 @@ Go to `http://localhost:4080/ <http://localhost:4080/>`_\<appname>/
 OR `http://localhost:8000/ <http://localhost:8000/>`_\<appname>/
 and you should see 'Welcome'.
 
+Configuring your app name and label
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+New in OMERO 5.3.0, we support the option of configuring your OMERO.web app with a
+name and label.
+See Django `Configuring Applications <https://docs.djangoproject.com/en/1.8/ref/applications/#configuring-applications>`_.
+This allows the URL to an app to be different from it's name.
```
Comment (simleo) [inline omero/developers/Web/CreateApp.txt]: "it's" -> "its"

File: omero/developers/whatsnew.txt [hunk 4]
```diff
@@ -2,12 +2,18 @@ What's new for OMERO 5.3 for developers
...
 
-Breaking changes
-----------------
-
 OMERO.web
 ^^^^^^^^^
 
+-  New ``api`` app to provide a new JSON API. See :doc:`/developers/json-api`
+-  Login functionality moved to Class-based LoginView in webgateway
+-  "Open with" config for opening apps from webclient. See :doc:`/developers/Web/LinkingFromWebclient'
```
Comment (hflynn) [inline omero/developers/whatsnew.txt]: You need a backtick at the end of this `:doc:` ref

**Comment by will-moore** [comment]
This is failing because it references json_api from #1627, so I'll just merge this branch into that one so everything is in a single PR.

**Pull request closed without merging**